apps/data_explorer/data_explorer.py

```diff
@@ -70,43 +88,49 @@ def construct_ui(blocks, datasets: Datasets,
         default_dataset = "ai_society_chat"
 
     misalignment_set_names = {"misalignment"}
-    ordinary_datasets = [
-        v for v in datasets.keys() if v not in misalignment_set_names
-    ]
-    misalignment_datasets = [
-        v for v in datasets.keys() if v in misalignment_set_names
-    ]
-    default_dataset_name = default_dataset \
-        if default_dataset in datasets.keys() \
-        else ordinary_datasets[0] if len(ordinary_datasets) > 0 \
-        else misalignment_datasets[0] if len(misalignment_datasets) > 0 \
+    ordinary_datasets = [v for v in datasets.keys() if v not in misalignment_set_names]
+    misalignment_datasets = [v for v in datasets.keys() if v in misalignment_set_names]
+    default_dataset_name = (
+        default_dataset
+        if default_dataset in datasets.keys()
+        else ordinary_datasets[0]
+        if len(ordinary_datasets) > 0
+        else misalignment_datasets[0]
+        if len(misalignment_datasets) > 0
         else ""
+    )
```

Comment:
Is the code block more readable as:

```suggestion
    # Determine the default dataset name
    if default_dataset in datasets:
        default_dataset_name = default_dataset
    elif ordinary_datasets:
        default_dataset_name = ordinary_datasets[0]
    elif misalignment_datasets:
        default_dataset_name = misalignment_datasets[0]
    else:
        default_dataset_name = ""
```